packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.controller.js

```diff
@@ -224,6 +246,37 @@ export default class AgoraIpV4OrderController {
     this.trackStep(3);
   }
 
+  onIpServiceSelection() {
+    this.isParkingIp =
+      this.model?.selectedService?.type === PRODUCT_TYPES.parking.typeName;
+    if (this.isParkingIp) {
+      this.loading.region = true;
+      this.parkingIpOffers = this.ipCatalog.filter((plan) =>
+        /^ip-v4|^ip-failover/.test(plan.planCode),
+      );
+      const DATACENTERS = this.parkingIpOffers
+        .map((ipOffer) => {
+          return ipOffer.details.product.configurations.find(
+            (config) => config.name === 'datacenter',
+          )?.values;
+        })
+        .flat();
+      const uniqueDatacenters = [...new Set(DATACENTERS)];
+      this.catalogByLocation = uniqueDatacenters.map((datacenter) => {
+        return {
+          datacenter,
```

Comment:
```suggestion
          datacenter,
          regionName: DATACENTER_TO_REGION[datacenter]
```
We should add region name (eu-west-rbx) within this object to display it on the UI

Reply:
done